### PR TITLE
Update Add vm extension command

### PR DESCRIPTION
**Pull request by cloudmelon**
Update add vm extension command with settings and protected settings

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Compute/Commands.Compute/help/Add-AzureRmVmssExtension.md [hunk 1]
```diff
@@ -30,8 +30,21 @@ The **Add-AzureRmVmssExtension** cmdlet adds an extension to the Virtual Machine
...
+This command adds an extension to the VMSS.
+
+### Example 2: Add an extension to the VMSS with settings and protected settings
+```
+PS C:\> $Settings = @{"fileUris" = "[]"; "commandToExecute" = ""};
+PS C:\> $ProtectedSettings = @{"storageAccountName" = $stoname; "storageAccountKey" = $stokey};
+
+PS C:\> Add-AzureRmVmssExtension -VirtualMachineScaleSet $vmss -Name $vmssExtensionName -Publisher $vmssPublisher  `
+  -Type $vmssExtensionType -TypeHandlerVersion $ExtVer -AutoUpgradeMinorVersion $True  `
+  -Setting $vmssSettings -ProtectedSetting $vmssProtectedSettings
```
Comment (cormacpayne) [inline src/ResourceManager/Compute/Commands.Compute/help/Add-AzureRmVmssExtension.md]: @cloudmelon should this line be `-Setting $Settings -ProtectedSetting $ProtectedSettings` ?

**Comment by cormacpayne** [comment]
@azuresdkci add to whitelist

**Comment by cloudmelon** [comment]
Yes exactly. Sorry I updated the definition but forget to update the variables. Thank you very much :)

Melony Qin

**Comment by cormacpayne** [comment]
@cloudmelon thanks for the contributions 😁

**Pull request merged**